index.html

```diff
@@ -5156,17 +5198,89 @@ <h3>Overview</h3>
         and generating a JSON-LD document in expanded form for all
         <a>RDF literals</a>, <a>IRIs</a>
         and <a>blank node identifiers</a>.
-        If the <a data-link-for="JsonLdOptions">useNativeTypes</a> flag is set to <code>true</code>,
-        <a>RDF literals</a> with a
-        <a>datatype IRI</a>
-        that equals <code>xsd:integer</code> or <code>xsd:double</code> are converted
-        to a <a>JSON numbers</a> and <a>RDF literals</a>
-        with a <a>datatype IRI</a>
-        that equals <code>xsd:boolean</code> are converted to <code>true</code> or
-        <code>false</code> based on their
-        <a>lexical form</a>
-        as described in
-        <a class="sectionRef" href="#data-round-tripping"></a>.
+
+        If the <a data-link-for="JsonLdOptions">useNativeTypes</a> flag is set
+        to <code>true</code>, <a>RDF literals</a> are converted to JSON values
+        based on their <a>datatype IRI</a> and <a>lexical form</a>, as described
+        in <a class="sectionRef" href="#algorithm-16"></a>.
+      </p>
+
+      <table class="simple" id="literal-conversation-with-useNativeTypes>
+        <caption>Literal conversion when <a data-link-for="JsonLdOptions">useNativeTypes</a> is set to <code>true</code></caption>
+        <thead>
+          <tr>
+            <th>RDF datatype</th>
+            <th>RDF Literal</th>
+            <th>JSON</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th><code>xsd:string</code></th>
+            <td>An RDF string literal</td>
+            <td>JSON string literal</td>
+          </tr>
+
+          <tr>
+            <th rowspan="3"><code>xsd:integer</code></th>
+            <td>Whole number in the <code>[-2<sup>53</sup> + 1, 2<sup>53</sup> + 1]</code> range, representable as per [[IEEE-754-2008]]</td>
```

Comment:
There are more constraints in the [lexical form of an xsd:integer](https://www.w3.org/TR/xmlschema11-2/#integer-lexical-representation) than this.

Also, while it's likely consistent with JCS/ECMAScript, such numbers also need to be representable as integers in JCS/ECMAScript. See [Appendix B](https://www.rfc-editor.org/rfc/rfc8785#json.ieee754.test). 
We should be careful to not mis-describe this, and just refer to XSD Datatypes 1.1 and JCS.